librz/include/rz_util/rz_ptrace.h

```diff
@@ -0,0 +1,63 @@
+// SPDX-FileCopyrightText: 2023 Florian Märkl <info@florianmaerkl.de>
+// SPDX-FileCopyrightText: 2017-2020 condret <condr3t@protonmail.com>
+// SPDX-FileCopyrightText: 2017-2020 pancake <pancake@nopcode.org>
+// SPDX-FileCopyrightText: 2017-2020 alvaro <alvaro.felipe91@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
```

Comment:
Maybe add a `\file` doxygen desription what is this file for? E.g. "contains abstractions on top of the target platform ptrace" or something like that?